ecmascript/transforms/compat/src/es2015/block_scoping.rs

```diff
@@ -936,14 +956,18 @@ impl VisitMut for MutationHandler<'_> {
 
     fn visit_mut_function(&mut self, n: &mut Function) {
         let old = self.in_function;
+        let arguments = self.arguments.clone();
```

Comment:
This can be changed to `self.arguments.take()`.
`Option.take()` takes the value (make target variable None) and returns the original value without cloning.